github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -230,7 +230,7 @@ presubmits:
         - /bin/sh
         - -c
         - cd cluster-provision/k8s/1.23 && ../provision.sh
-        image: quay.io/kubevirtci/golang:v20220211-d7d6c59
+        image: quay.io/kubevirtci/golang:podman-test21
```

Comment:
we do not want to take this image in - it was added for testing https://github.com/kubevirt/project-infra/pull/2055

all of the golang images have this `podman-test21`